src/frontend/js/widgets/Dashboard/components/DashboardItem/index.tsx

```diff
@@ -66,3 +81,28 @@ export const DashboardItem = ({
     </div>
   );
 };
+
+const DashboardItemMore = ({ more }: { more: ReactNode }) => {
```

Comment:
This more menu should be available for reuse, it could be in `js/components` with it's `MoreMenuItem`